org.lflang/src/org/lflang/LinguaFranca.xtext

```diff
@@ -251,12 +251,12 @@ Serializer:
 /////////// For target parameters
 
 KeyValuePairs:
-    '{' pairs+=KeyValuePair (',' (pairs+=KeyValuePair))* ','? '}';
+    '{' ( pairs+=KeyValuePair (',' (pairs+=KeyValuePair))* )? ','? '}';
```

Comment:
```suggestion
    {KeyValuePairs} '{' (pairs+=KeyValuePair (',' (pairs+=KeyValuePair))* ','?)? '}';
```
I'm OK with a trailing comma for convenience, but `{ , }` just looks sloppy. This grammar rule accepts `{ }` and `{ timeout: 3 sec, }` but not `{ , }`.

Reply:
Ok I agree `{,}` should be avoided, but what does this `{KeyValuePairs}` mean?

Reply:
It's an explicit "action" that forces the creation of an empty pairs object if no pairs were found.

Reply:
Try it out :-)

Reply:
Very cool!! This will actually help with #544